k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -780,8 +754,10 @@ function waitForClusterReady()
     fi
 
     if [[ ${components[data]} == true ]]; then
-        (waitForAllDeploymentsAvailable "${CORTX_DEPLOY_DATA_TIMEOUT:-10m}" statefulset/cortx-data) &
-        pids+=($!)
+        for statefulset in $(kubectl get statefulset -l app.kubernetes.io/component=data,app.kubernetes.io/instance=cortx --no-headers | awk '{print $1}'); do
```

Comment:
```suggestion
        kubectl get statefulset --selector app.kubernetes.io/component=data,app.kubernetes.io/instance=cortx --no-headers --output custom-columns=NAME:metadata.name \
          | while read -r statefulset; do
```
**suggestion:** Could do all this in bash w/o external tools. (Or template/jsonpath, etc).

Also prefer using long options for self-documentation.